lib/utils.pm

```diff
@@ -957,7 +957,13 @@ sub ensure_serialdev_permissions {
     # ownership has effect immediately, group change is for effect after
     # reboot an alternative https://superuser.com/a/609141/327890 would need
     # handling of optional sudo password prompt within the exec
-    assert_script_run "chown $testapi::username /dev/$testapi::serialdev && gpasswd -a $testapi::username \$(stat -c %G /dev/$testapi::serialdev)";
+    # Need backwards supporr for SLES11-SP4 here, the command "gpasswd" and "stat" are only be availabled with SLES-12 at latest.
```

Comment:
Please fix the typos here: supporr (support), are only be availabled (are only available), at latest (at least)